src/tqec/plaquette/rpng_description.py

```diff
@@ -0,0 +1,245 @@
+from tqec.circuit.schedule import ScheduledCircuit
+from tqec.circuit.qubit_map import QubitMap
+from tqec.plaquette.plaquette import Plaquette
+from tqec.plaquette.qubit import PlaquetteQubits
+from tqec.plaquette.qubit import SquarePlaquetteQubits
+
+from dataclasses import dataclass
+from enum import Enum
+from typing import Optional
```

Comment:
We can use `A | None` instead of `Optional[A]` for consistency in the codebase.

Reply:
Applied.